tensorboard/plugins/interactive_inference/witwidget/notebook/visualization.py

```diff
@@ -363,10 +371,77 @@ def set_compare_estimator_and_feature_spec(self, estimator, feature_spec):
     Returns:
       self, in order to enabled method chaining.
     """
+    # If custom function is set, remove it before setting estimator
+    self.delete('custom_predict_fn')
```

Comment:
i think this should only delete the custom compare predict fn

Reply:
done